python/pyspark/streaming/listener.py

```diff
@@ -23,6 +23,12 @@ class StreamingListener(object):
     def __init__(self):
         pass
 
+    def onStreamingStarted(self, streamingStarted):
```

Comment:
Although in #21057, it is mentioned that there will be an exception to be thrown if this method is lack. However, I can't reproduce such error.

Reply:
Because `PythonStreamingListener` have such method, I think it is still better to add it here. And I also add a test to make sure it works.